e2e/scripts/st_arrow_dataframe_canvas_rendering.py

```diff
@@ -1,341 +0,0 @@
-# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
```

Comment:
This script was refactored into multiple more specific e2e test scripts which all together cover a lot more aspects.

Reply:
Also, this script was (most of the time) the slowest e2e script because of all the themed snapshot tests which slowed down our CI pipeline. The new split-up should fix this.